metadata-ingestion/src/datahub/ingestion/source/usage/redshift_usage.py

```diff
@@ -223,7 +223,18 @@ def _get_redshift_history(
                 event_dict["endtime"] = event_dict.get("endtime").__str__()
 
             logger.debug(f"event_dict: {event_dict}")
-            events.append(event_dict)
+            # filter based on schema and table pattern
+            if (
+                event_dict["schema"]
+                and self.config.schema_pattern.allowed(event_dict["schema"])
+                and event_dict["table"]
+                and self.config.table_pattern.allowed(event_dict["table"])
+            ):
+                events.append(event_dict)
+            else:
+                logger.debug(
```

Comment:
Please add this to the source report as well. See [this example] (https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py#L187)

Reply:
Hey @rslanka, I have added this to the reporting as well